src/runtime/int.cpp

```diff
@@ -1157,30 +1212,64 @@ void setupInt() {
                                                                                    ParamNames::empty(), CAPI)));
 
     _addFuncIntFloatUnknown("__add__", (void*)intAddInt, (void*)intAddFloat, (void*)intAdd);
```

Comment:
If I rewrite the `__add__` implementation like `__div__`. The test suite will report `binops_ics` will report `exception: ("noninit_count('slowpath_binop') <= 10", 'slowpath_binop', 1005)` error.